x-pack/plugin/apm-data/src/main/resources/component-templates/apm@mappings.yaml

```diff
@@ -17,3 +17,7 @@ template:
         type: constant_keyword
       data_stream.namespace:
         type: constant_keyword
+      event.ingested:
+        type: date
+        format: "strict_date_time_no_millis||strict_date_optional_time||epoch_millis"
+        ignore_malformed: false
```

Comment:
do we need to redefine this, when it's already part of ECS, so imported through `ecs@mappings`?

Reply:
I don't think we have to, unless you want to use an indexed runtime field like we have in the trace.sampled data stream: https://github.com/elastic/elasticsearch/blob/4e8e9731567d66ed4cb7c3a2c7ed5af744e32ae7/x-pack/plugin/apm-data/src/main/resources/index-templates/traces-apm.sampled%40template.yaml#L33-L36

Reply:
ok thanks i'll remove it